### DISCUSSION TITLE
WebsiteFilter_Exceptions - Multiple entries

### DISCUSSION THREAD

**Discussion by Niglb** [mozilla/policy-templates#871] [Q&A]
Hi there, 

Just trying to confirm the syntax for multiple WebsiteFilter_Exceptions Values - from your example, how would one do this in Intune?

**Comment by mkaply** [comment]
For multiple values, it looks like this:
```
<data id="WebsiteFilter" value="1&#xF000;http://example.org/*&#xF000;2&#xF000;http://example.edu/*"/>
```

From the Microsoft docs:

"Note that it is expected that each string in the SyncML is to be separated by the Unicode character 0xF000 (encoded version: &#xF000;)"

**Reply by Niglb** [reply]
interesting ok I am just trying to figure out the syntax for more than on URL.

**Comment by mkaply** [comment]
Sorry about that. Didn't notice that git ate my content :)

**Reply by mkaply** [reply]
(What I meant by that comment is take a look at my post again. It has the syntax now)
